include/h2o.h

```diff
@@ -321,6 +322,13 @@ typedef struct st_h2o_status_handler_t {
 } h2o_status_handler_t;
 
 typedef H2O_VECTOR(h2o_status_handler_t) h2o_status_callbacks_t;
+
+typedef enum h2o_forward_early_hint {
```

Comment:
should be "early hints" throughout the PR, especially when dealing with a set of header fields. This is because each header field of an 103 response is a "hint".